src/commands/dev/lib/dev-server.ts

```diff
@@ -79,11 +80,27 @@ export default class DevServer {
     this.statusMessage = msg;
   }
 
+  async getNowJson(): Promise<NowConfig | null> {
+    const nowJsonPath = getNowJsonPath(this.cwd);
+    console.error({ nowJsonPath });
+
+    try {
+      return JSON.parse(await fs.readFile(nowJsonPath, 'utf8'));
+    } catch (err) {
+      if (err.code === 'ENOENT') {
+        return null;
+      } else {
+        throw err;
+      }
+    }
+  }
+
   /**
    * Launch dev-server
    */
   async start(port: number = 3000): Promise<void> {
-    const nowJson = readLocalConfig(this.cwd);
+    const nowJson = await this.getNowJson();
```

Comment:
@TooTallNate Did you create this custom `getNowJson` to avoid those `process.exit(1)` in [`readLocalConfig`](https://github.com/zeit/now-cli/blob/5a111186f4a47e0b27d4ee1de9c365f9da6baf11/src/util/config/files.ts#L71-L98)?

Reply:
The plan is that we can optimize this function to cache the result, and also add validation (i.e. for `version: 2`). It will then be invalidated with a watcher / periodic shasum / whatever mechanism we decide on to reduce the number of times the file is being read + parsed.